edgedb/_testbase.py

```diff
@@ -393,7 +393,7 @@ def setUp(self):
         if self.INTERNAL_TESTMODE:
             self.loop.run_until_complete(
                 self.client.execute(
-                    'CONFIGURE SESSION SET __internal_testmode := true;'))
+                    'CONFIGURE INSTANCE SET __internal_testmode := true;'))
```

Comment:
It wasn't `SYSTEM` it's `SESSION`. Are you sure this has to be changed?

Reply:
Oh, whoops. This change is, indeed, incorrect